test/index.js

```diff
@@ -116,6 +116,20 @@ describe('PWMetrics', () => {
         });
       });
     });
+
+    describe('with one run with expectations', () => {
+      beforeEach(() => {
+        const {opts} = runOptions.startWithOneRunWithExpectations;
+        pwMetrics = new PWMetrics(runOptions.startWithOneRun.url, opts);
```

Comment:
Not in that case, because if you look into the fixtures, there are some cases covering when the configuration has expectations without configuration. That was the reason to create a new one instead use an existent one.